### ISSUE TITLE
Unable to install from URL with mixed-case

### ISSUE THREAD

**Issue by forsberg** [TheKevJames/puppet-homebrew#105]
For.. reasons, I need a specific version of kubernetes-cli installed. I use this manifest:

```
package {
    # kubernetes-cli (kubectl) 1.8.6
    'https://raw.githubusercontent.com/Homebrew/homebrew-core/e5538d7e7966795ae50cd94678ee7131fad2d7b7/Formula/kubernetes-cli.rb':
      notify => Exec['pin-kubernetes-cli'],
      before => Package['kops'],
      ensure => present;
}

However, puppet-homebrew lowercases the resource name before passing it to `brew install`, and with the above URL, that obviously leads to a 404 and failure. Argh.

Perhaps there's a better way of doing this?

**Comment by TheKevJames** [comment]
@forsberg I just opened PR #110 to support this -- it seems to be a fairly simple change, but I don't actually own a Mac anymore for testing purposes. Could you verify that branch fixes your issue?

**Comment by TheKevJames** [comment]
Fixed and released in version 1.8.3, now available [on the Puppet Forge](https://forge.puppet.com/thekevjames/homebrew)
